set.mm

```diff
@@ -29962,9 +29964,11 @@ Such interpretation is rarely needed (see also ~ df-ral ).  (Contributed
        quantifier with implicit substitution which also changes the quantifier
        domain.  Deduction form.  (Contributed by David Moews, 1-May-2017.) $)
     cbvrexdva2 $p |- ( ph -> ( E. x e. A ps <-> E. y e. B ch ) ) $=
-      ( cv wcel wa wex wrex weq simpr eleq12d anbi12d cbvexdva df-rex 3bitr4g )
-      ADJZFKZBLZDMEJZGKZCLZEMBDFNCEGNAUDUGDEADEOZLZUCUFBCUIUBUEFGAUHPIQHRSBDFTC
-      EGTUA $.
+      ( cv wcel wa wex wrex wn wal wi 19.21v alnex df-rex simpr eleq12d anbi12d
+      wb notbid expcom pm5.74d cbvalvw 3bitr3i pm5.74ri 3bitr3g con4bid 3bitr4g
+      weq ) ADJZFKZBLZDMZEJZGKZCLZEMZBDFNCEGNAURVBAUQOZDPZVAOZEPZUROVBOAVDVFAVC
```

Comment:
so mainly, cbvexdva was replaced by cbvalvw. Maybe there is a shorter proof using cbvexvw ?

Reply:
I tested with the minimizer under various conditions and didn't found anything, there might be some possible manual shortenings, but my confidence with the database is not that great yet.

Reply:
Well, the ultimate minimizer is called @wlammen ! So if you have time/interest, Wolf, you may have a look at that one.